packages/wallets/src/features/cfd/constants.tsx

```diff
@@ -1,30 +1,31 @@
-import React, { ReactNode } from 'react';
+import React from 'react';
 import DerivedMT5Icon from '../../public/images/mt5-derived.svg';
 import FinancialMT5Icon from '../../public/images/mt5-financial.svg';
 import SwapFreeMT5Icon from '../../public/images/mt5-swap-free.svg';
-import { TMarketTypes, TPlatforms } from '../../types';
+import { TPlatforms } from '../../types';
 
-export const MarketTypeToTitleMapper: Record<TMarketTypes.All, string> = {
-    all: 'Swap-Free',
-    financial: 'Financial',
-    synthetic: 'Derived',
-};
+export const MarketTypeDetails = {
+    all: {
+        description:
+            'Trade swap-free CFDs on MT5 with synthetics, forex, stocks, stock indices, cryptocurrencies and ETFs',
+        icon: <SwapFreeMT5Icon />,
```

Comment:
Question: isn't is better to write 
`icon: () => <SwapFreeMT5Icon />`
and then it can be used as <Icon /> where the configuration is being used?

Reply:
it's mostly the same. u can just {icon} (without the need of capitalization) and it will transform to jsx since the variable is already in jsx